src/org/ods/orchestration/service/JiraService.groovy

```diff
@@ -763,4 +761,36 @@ class JiraService {
         return true
     }
 
+    @NonCPS
+    Map getComponentsStatus(String projectKey, String version) {
```

Comment:
Please, rename this to getComponents. If there is any conflict, we can discuss it.